pkg/iris/api_calls.go

```diff
@@ -0,0 +1,56 @@
+package iris
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+const IRIS_URI string = "https://eu.leanix.net/services/vsm-iris/v1/"
+
+func GetConfiguration(configurationName string, accessToken string) ([]byte, error) {
+	configUrl := fmt.Sprintf(IRIS_URI+"configurations/kubernetesConnector/%s", configurationName)
+	req, err := http.NewRequest("GET", configUrl, nil)
+	req.Header.Set("Authorization", "Bearer "+accessToken)
+	if err != nil {
+		log.Infof("SelfStartRun: Error while retrieving configuration from %s: %v", configUrl, err)
+		return nil, err
+	}
+	resp, err := http.DefaultClient.Do(req)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
```

Comment:
I would suggest we restructure the packages a little bit.

Similar to Kubernetes we can have the iris package in /pkg but then we should have also have an api.go which is a struct which implements a new interface which has the GetCofiguration() and PostResults() functions.
It should be very similar to the api.go in the Kubernetes package.

Reply:
agree, thats a good Idea! I will change it.